static/app/components/searchSyntax/parser.tsx

```diff
@@ -1362,6 +1371,10 @@ export type SearchConfig = {
    * Text filter keys we allow to have operators
    */
   textOperatorKeys: Set<string>;
+  /**
+   * When true, the parser will check for the contains flag in text values
+   */
+  enableContainsCheck?: boolean;
```

Comment:
I assume that startsWith and endsWith will be added in the future, so I'm wondering if maybe you want this to be more generic like `parseWildcards` or something?

Reply:
Good idea 🧠 will make those changes.